frontend/src/authentication/components/AuthGuard.js

```diff
@@ -1,18 +1,59 @@
 import PropTypes from 'prop-types';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { Navigate, useLocation } from 'react-router-dom';
 import { Login } from '../views/Login';
 import { useAuth } from '../hooks';
 import {
+  isModuleEnabled,
+  ModuleNames,
   RegexToValidateWindowPathName,
   WindowPathLengthThreshold
 } from '../../utils';
```

Comment:
All these relative imports from line 11 to 19, both included, should be absolute imports (even the one in line 11 which was introduced in this PR).

Reply:
Updated imports for `utils`, `design`, `globalErrors`. For others like `../../modules/Maintenance/services` & `../../modules/Maintenance/views/MaintenanceViewer` I don't know if these can be absolute imports . Can you tell me if you were asking for these to also be absolute imports ?

Reply:
This is a very particular case, because in general frontend modules do not influence shared components, but this is a banner on the whole screen. 

**OPTION 1:** maintenance as part of the shared elements in FE
- move whatever is in `modules/Maintenance/services' to `services`
- move `modules/Maintenance/components/MaintenanceViewer` to `design/components`

We import services and viewer with absolute imports in
- frontend/src/authentication/components/AuthGuard.js
- frontend/src/modules/Administration/views/AdministrationView.js


**OPTION 2**: maintenance as module in FE modules
We keep it as it is now.


Downside of OPTION 1, for a new developer it is more difficult to find Maintenance code. But there is a reason why in the FE we enforce absolute imports all over the place, it has to do with the way the FE modularization is intended. 
The idea is that all packages except for `modules` expose some methods that all the other packages can use, the methods exposed are chosen in the index.js. Code that is not in the index should not be used by other packages. This way working inside each of the packages sets a boundary of the responsibilities "this are the functions in the index.js anything else I can touch as I please".

`modules` use the global packages and develop additional code inside the module. **Removing one module should not break anything!** Right now if we remove the Maintenance module there will be errors, that's why I think we should go for OPTION 1